packages/quantic/force-app/examples/main/lwc/actionSelectTab/actionSelectTab.html

```diff
@@ -0,0 +1,13 @@
+<template>
+  <div class="slds-grid slds-gutters">
+    <div class="slds-col">
+      <lightning-button
```

Comment:
Self-closing tag?

Reply:
I think we don’t use self-colosing tags with custom components to respect the HTML standard documented [here](https://html.spec.whatwg.org/multipage/parsing.html#parse-error-non-void-html-element-start-tag-with-trailing-solidus)